packages/devextreme-scss/scss/widgets/fluent/chat/_sizes.scss

```diff
@@ -0,0 +1,49 @@
+@use "../sizes" as *;
+
+// adduse
+
+$chat-bubble-border-radius: null !default;
```

Comment:
I didn't check all values in all themes to ensure they match the design. Let's address this in the second iteration of the review